File: eFormAPI/eFormAPI.Web/Infrastructure/Models/SearchableList/AdvEntitySearchableGroupListRequestModel.cs
namespace Microting.eFormApi.BasePn.Infrastructure.Models.SearchableList
{
    public class AdvEntitySearchableGroupListRequestModel
    {
        public string Sort { get; set; }
        public string NameFilter { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }
        public bool IsSortDsc { get; set; }
    }
}

File: eform-client/src/app/common/models/advanced/searchable-groups/adv-entity-searchable-group-list-request.model.ts
export class AdvEntitySearchableGroupListRequestModel {
  public sort: string;
  nameFilter: string;
  pageIndex: number;
  pageSize: number;
  isSortDsc: boolean;
  offset: number;

  constructor() {
    this.sort = '';
    this.nameFilter = '';
    this.pageSize = 5;
    this.pageIndex = 0;
    this.offset = 0;
  }
}
